services/app-api/src/handlers/bulk_download.ts

```diff
@@ -82,21 +86,25 @@ export const main: APIGatewayProxyHandler = async (event) => {
     const s3DownloadStreams: S3DownloadStreamDetails[] = await Promise.all(
         bulkDlRequest.keys.map(async (key: string) => {
             const params = { Bucket: bulkDlRequest.bucket, Key: key }
-            const metadata = await s3.headObject(params).promise()
+            const headCommand = new HeadObjectCommand(params)
+            const metadata = await s3.send(headCommand)
+            const getCommand = new GetObjectCommand(params)
+            const filename = parseContentDisposition(
+                metadata.ContentDisposition ?? key
+            )
+            const s3Item = await s3.send(getCommand)
             return {
-                stream: s3.getObject(params).createReadStream(),
+                stream: s3Item.Body as Readable,
```

Comment:
This is type coercion which could be _not a good idea_
Based this solution on https://transang.me/modern-fetch-and-how-to-get-buffer-output-from-aws-sdk-v3-getobjectcommand/ 
^ that article does have good context on the V3 get object return types